modules/javafx.controls/src/test/java/test/com/sun/javafx/scene/control/infrastructure/MouseEventFirerTest.java

```diff
@@ -179,18 +191,15 @@ protected void assertMouseEventCoordinates(MouseEvent mouse) {
         String text = receiver instanceof Labeled ? ((Labeled) receiver).getText() : receiver.getId();
 
         Point2D sceneP = receiver.localToScene(mouse.getX(), mouse.getY());
-        assertEquals("sceneX of " + text, sceneP.getX(), mouse.getSceneX(), EPS);
-        assertEquals("sceneY of " + text, sceneP.getY(), mouse.getSceneY(), EPS);
+        assertEquals(sceneP.getX(), mouse.getSceneX(), EPS, "sceneX of " + text);
+        assertEquals(sceneP.getY(), mouse.getSceneY(), EPS, "sceneY of " + text);
         Point2D screenP = receiver.localToScreen(mouse.getX(), mouse.getY());
-        assertEquals("screenX of " + text, screenP.getX(), mouse.getScreenX(), EPS);
-        assertEquals("screenY of " + text, screenP.getY(), mouse.getScreenY(), EPS);
+        assertEquals(screenP.getX(), mouse.getScreenX(), EPS, "screenX of " + text);
+        assertEquals(screenP.getY(), mouse.getScreenY(), EPS, "screenY of " + text);
     }
 
  // ------------- parameterized in not/alternative mouseEvent creation
 
-    private boolean useAlternative;
-
-    @Parameterized.Parameters
     public static Collection<Object[]> data() {
```

Comment:
Minor: Could also be simplified with the argument stream

Reply:
this was fixed in one of the subsequent commits.